tests/e2e/test_main.py

```diff
@@ -425,19 +445,25 @@ def test_image_transfer(
                 )
             )
 
-        result = cli_runner(
-            ["neuro", "image-build", "-f", str(dockerfile_path), ".", from_img]
-        )
-        assert result.returncode == 0, result
+        try:
+            result = cli_runner(
+                ["neuro", "image-build", "-f", str(dockerfile_path), ".", from_img]
+            )
+            assert result.returncode == 0, result
 
-        result = repeat_until_success(["neuro", "image", "tags", f"image:{img_name}"])
-        assert tag in result.stdout
+            result = repeat_until_success(
```

Comment:
We had a `sleep(10)` call in other cases when the image was built, don't we need it here? (or, maybe, it is redundant in all other places, which might speedup tests).